lua/autorun/server/sv_autoclean.lua

```diff
@@ -9,13 +9,13 @@ local clientCleanupCommands = {
     ["stopsound"] = true
 }
 
-local function notifyPlayers( notification )
-    local message = "[CFC_Autoclean] " .. notification
-
-    print( message )
+local function notifyPlayers( message, includePrefix )
+    local prefix = "[CFC - AutoClean] "
+    local printMsg = includePrefix and prefix .. message or message
```

Comment:
how about we make the prefix opt-out instead of opt-in

```suggestion
local prefix = includePrefix != false and prefix or ""
local printMsg = prefix .. message
```

This way you can call `notifyPlayers( "My Message" )` and get the default prefix, or `notifyPlayers( "[CFC-Custom] My Message", false )` to exclude it.

I also think `printMsg` should be renamed to something like `notification`

Reply:
`~=` ?

Reply:
🤮

Reply:
Wouldn't `includePrefix` be nil instead of false?

Reply:
If it wasn't included, yes, that's why I explicitly compared with `false`

that way if you don't pass the argument, it still includes the prefix. you'll have to specifically pass `false` to prevent the thing from showing

Reply:
I would have guessed it would be nil, but alright I'll get to it.

Reply:
Wouldn't `nil` be not equal to false thus putting us in the same scenario if we we to include the `false` at the end of the function...?

Reply:
A `nil` check sounds less confusing to me, this is hurting my head xD

Reply:
It needs to be a `false` check to work the way I want.


Scenarios:

1. `notifyPlayers( "Message" )`
Second parameter is `nil`, so it would result in this:
```lua
    local prefix = includePrefix != false and prefix or ""
    -- ( nil != false and prefix ) or ""
    -- ( ( true and prefix ) or "" ) results in 'prefix` 

    local printMsg = prefix .. message
    -- results in "[CFC_Autoclean]" .. "Message"
```

2. `notifyPlayers( "Message", true )`
Would result in:
```lua
    local prefix = includePrefix != false and prefix or ""
    -- ( true != false and prefix ) or ""
    -- ( ( true and prefix ) or "" ) results in `prefix`

    local printMsg = prefix .. message
    -- results in "[CFC_Autoclean]" .. "Message"
```

3. `notifyPlayers( "Message", false )`
Would result in:
```lua
    local prefix = includePrefix != false and prefix or ""
    -- ( false != false and prefix ) or ""
    -- ( ( false and prefix ) or "" ) results in ""

    local printMsg = prefix .. message
    -- results in "" .. "Message"
```


Put simply; if you pass `false` as the second parameter, it will not include the default prefix. Any other value, or lack thereof, will use the default prefix.

Reply:
Ok, was just having troubles with the truth tables and such. A literal brain fart

Reply:
You bring up a good point, maybe it should be broken up into multiple checks for clarity.

Reply:
```lua
local printMsg = message
if includePrefix ~= false then 
    printMsg = printMsg .. prefix 
end
``` 
yeah, maybe something like this

Reply:
alright gang we finally cracked the case